BootloaderCommonPkg/Library/FatLib/FatLib.c

```diff
@@ -12,6 +12,38 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include "FatLiteApi.h"
 #include "FatLiteFmt.h"
 
+/**
+  Print directory or file name in debug output
+
+  @param[in]     File             The File pointer to be printed
+
+  @retval        none
+
+*/
+STATIC
+VOID
+PrintFileName (
+  IN   PEI_FAT_FILE  *File
+  )
+{
+  CHAR16  *FileName;
+
+  if (File == NULL) {
+    return;
+  }
+
+  FileName = File->LongFileName;
+  if (FileName[0] == '\0') {
+    FileName = File->FileName;
+  }
+
+  if (File->Attributes & FAT_ATTR_DIRECTORY) {
+    DEBUG ((DEBUG_INFO, "  %s/\n", FileName));
+  } else {
+    DEBUG ((DEBUG_INFO, "  %s\t%d\n", FileName, File->FileSize));
```

Comment:
Let us use fixed width for filename so that the file size can be aligned in the print.

Reply:
Sure I will change this to use fixed width.

Reply:
Fixed 16chars file name is used in 3rd patch. This function will be changed later when supporting display console. Just FYI.